RestaurantReservation.API/Services/JwtTokenGenerator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using RestaurantReservation.API.Interfaces;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace RestaurantReservation.API.Services
+{
+    public class JwtTokenGenerator : IJwtTokenGenerator
+    {
+
+        private readonly IConfiguration _configuration;
+
+        public JwtTokenGenerator(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public string GenerateToken(string UserName, string Password)
+        {
+            var claims = new[]
+            {
+               new Claim(ClaimTypes.Name, UserName),
+               new Claim("Password", Password)
```

Comment:
why are you adding password in claims ? won't that be exposing sensitive information